src/Organisation/OrganisationService.php

```diff
@@ -359,7 +359,7 @@ public function getOrganisationFromUserId(int $userId) : Organisation
      *
      * @param string $organisationId
      *
-     * @return mixed|null
+     * @return \ArrayIterator
```

Comment:
juste \Iterator, comme ça si on change d'itérateur plus tard, avec de la pagination, ça sera tjs bon